ethkey/main.cpp

```diff
@@ -82,23 +86,48 @@ int main(int argc, char** argv)
 	setDefaultOrCLocale();
 	KeyCLI m(KeyCLI::OperationMode::ListBare);
 	g_logVerbosity = 0;
-
+	po::options_description generalOptions("General Options");
+	generalOptions.add_options()
+		("verbosity,v", po::value<string>(), "<0 - 9>  Set the log verbosity from 0 to 9 (default: 8).")
+		("version,V", "Show the version and exit.")
+		("help,h",  "Show this help message and exit.")
+	;
```

Comment:
This formatting looks weird in our style, please leave semicolon on the last line of the statement.